engine/src/main/java/org/hibernate/validator/internal/constraintvalidators/bv/money/DigitsValidatorForMonetaryAmount.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Hibernate Validator, declare and validate application constraints
+ *
+ * License: Apache License, Version 2.0
+ * See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
+ */
+package org.hibernate.validator.internal.constraintvalidators.bv.money;
```

Comment:
@gsmet I saw you mentioning hv/bv packages on the old money related PR. It was probably for this one though :smiley: 
I'd keep this one in bv. This way we would have all money related validators in one package.

Reply:
Yeah, it was for this PR, not sure how I ended up on the other one.

In this case, I don't think consistency is a good idea: it's important to know that these validators are not part of Bean Validation. So that we can think about moving them to the spec at some point.

Reply:
OK, I took a closer look and, AFAICS, as long as it is a BV constraint, the validators are in the `bv` package.

So let's leave it at that. Even if I'm not entirely convinced it's a good idea.